pep-0639.rst

```diff
@@ -550,339 +2227,375 @@ The core metadata documentation ``License`` field documentation is currently::
 
-- Rust Cargo [#cargo]_ specifies the use of an SPDX license expression (v2.1) in
-  the ``license`` field. It also supports an alternative expression syntax using
-  slash-separated SPDX license identifiers. There is also a ``license_file``
-  field. The crates.io package registry [#cratesio]_ requires that either
-  ``license`` or ``license_file`` fields are set when you upload a package.
-
-- PHP Composer composer.json [#composer]_ uses a ``license`` field with an SPDX
-  license id or "proprietary". The ``license`` field is either a single string
-  that can use something which resembles the SPDX license expression syntax with
-  "and" and "or" keywords; or is a list of strings if there is a choice of
-  licenses (aka. a "disjunctive" choice of license).
-
-- NuGet packages [#nuget]_ were using only a simple license URL and are now
-  specifying to use an SPDX License expression and/or the path to a license
+- `Rust Cargo <#cargo_>`_ specifies the use of an SPDX license expression
+  (v2.1) in the ``license`` field. It also supports an alternative expression
+  syntax using slash-separated SPDX license identifiers, and there is also a
+  ``license_file`` field. The `crates.io package registry <#cratesio_>`_
+  requires that either ``license`` or ``license_file`` fields are set when
+  uploading a package.
+
+- `PHP Composer composer.json <#composer_>`_ uses a ``license`` field with
+  an SPDX license ID or ``proprietary``. The ``license`` field is either a
+  single string with resembling the SPDX license expression syntax with
+  ``and`` and ``or`` keywords; or is a list of strings if there is a
+  (disjunctive) choice of licenses.
+
+- `NuGet packages <#nuget_>`_ previously used only a simple license URL, but
+  now specify using a SPDX license expression and/or the path to a license
   file within the package. The NuGet.org repository states that they only
-  accepts license expressions that are `approved by the Open Source Initiative
-  or the Free Software Foundation.`
+  accept license expressions that are "approved by the Open Source Initiative
+  or the Free Software Foundation."
 
 - Go language modules ``go.mod`` have no provision for any metadata beyond
   dependencies. Licensing information is left for code authors and other
   community package managers to document.
 
-- Dart/Flutter spec [#flutter]_ recommends to use a single ``LICENSE`` file
-  that should contain all the license texts each separated by a line with 80
-  hyphens.
+- The `Dart/Flutter spec <#flutter_>`_ recommends using a single ``LICENSE``
+  file that should contain all the license texts, each separated by a line
+  with 80 hyphens.
 
-- JavaScript Bower [#bower]_ ``license`` field is either a single string or a list
-  of strings using either SPDX license identifiers, or a path or a URL to a
-  license file.
+- The `JavaScript Bower <#bower_>`_ ``license`` field is either a single or
+  or list of strings using either SPDX license identifiers, or a path/URL
```

Comment:
I dropped `string` to avoid repetition, but accidentally adding a more egregious repetition of a double "or" here...d'oh! And for the interests of clarity, I don't mind having `string` back.

Thanks to this comment, I ran a quick regex (`\b(\w{2,})\n? +\1\b`) to check for instances of double words, and actually found a few more, which I also fixed. Thanks!